src/libsyntax/feature_gate.rs

```diff
@@ -1083,7 +1083,8 @@ pub const BUILTIN_ATTRIBUTES: &[(&str, AttributeType, AttributeTemplate, Attribu
                                               stable",
                                              cfg_fn!(profiler_runtime))),
 
-    ("allow_internal_unstable", Normal, template!(Word), Gated(Stability::Unstable,
+    ("allow_internal_unstable", Normal, template!(Word, List: "feat1, feat2"),
```

Comment:
```suggestion
    ("allow_internal_unstable", Normal, template!(Word, List: "feat1, feat2, ..."),
```
For consistency with other diagnostics.